src/Compilers/CSharp/Portable/Syntax/SyntaxNormalizer.cs

```diff
@@ -337,10 +364,14 @@ private static int LineBreaksAfterOpenBrace(SyntaxToken currentToken, SyntaxToke
 
         private static int LineBreaksAfterCloseBrace(SyntaxToken currentToken, SyntaxToken nextToken)
         {
-            if (currentToken.Parent is InitializerExpressionSyntax or SwitchExpressionSyntax or PropertyPatternClauseSyntax ||
-                currentToken.Parent.IsKind(SyntaxKind.Interpolation) ||
-                currentToken.Parent?.Parent is AnonymousFunctionExpressionSyntax ||
-                IsAccessorListFollowedByInitializer(currentToken.Parent))
+            var currentTokenParent = currentToken.Parent;
+            if (currentTokenParent is SwitchExpressionSyntax or PropertyPatternClauseSyntax ||
+                currentTokenParent.IsKind(SyntaxKind.Interpolation) ||
+                currentTokenParent?.Parent is AnonymousFunctionExpressionSyntax ||
+                IsAccessorListFollowedByInitializer(currentTokenParent) ||
+                isCloseBraceFollowedByComma(currentToken, nextToken) ||
+                nextToken.Parent is MemberAccessExpressionSyntax or BracketedArgumentListSyntax ||
```

Comment:
nit: consider sandwiching comments to make all these cases easier to follow. For instance, this is for `{...}.Length` and `new []{...}[0]` scenarios

Reply:
Not sure what you mean by "sandwiching comments" (should I place them between these lines or after and if after then should I align them?). I did the "after each line without alignment" case, feel free to correct me here)

Reply:
Thanks\